features/feature-policy.yml

```diff
@@ -0,0 +1,55 @@
+name: Feature policy
+description: The `Feature-Policy` response header sets whether a policy-controlled feature, such as an API, may be used in a document. Not to be confused with permissions policy.
```

Comment:
The `allow` attributes on iframes was also an important part of the original feature policy. That's now defined by HTML and permissions policy. This feature would have been described differently at the time it was shipped, but we should update description as the platform evolves so this is OK.